UnityProject/Assets/Scripts/Shuttles/MatrixMove.cs

```diff
@@ -177,6 +177,9 @@ public class MatrixMove : ManagedNetworkBehaviour
 	/// </summary>
 	public bool Initialized => clientStarted && receivedInitialState;
 
+	[Tooltip("Disable the ability for players to use a shuttleconsole to control this matrix")]
+	public bool NoConsole = false;
```

Comment:
Although the tooltip is great, maybe this name is not the best to understand what it does at first glance. Change it to something like ``IsPilotable``. You would need to invert the logic check if you choose that name.